dietpi/dietpi-software

```diff
@@ -15704,6 +15712,8 @@ _EOF_
 		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == -1 )); then
 
 			Banner_Uninstalling
+
+			G_EXEC ln -s /mnt/dietpi_userdata/postgresql /var/lib/postgresql/11/main
```

Comment:
On Bullseye, it will be `/var/lib/postgresql/13/main`, on Stretch it's `/var/lib/postgresql/9/main`, let's:
```sh
G_EXEC ln -s /mnt/dietpi_userdata/postgresql /var/lib/postgresql
```
And to avoid failures on reinstalls or when the package was manually installed before, we need to move the directory, if it exists already and remove the link, if it exists already. I'll push a commit.